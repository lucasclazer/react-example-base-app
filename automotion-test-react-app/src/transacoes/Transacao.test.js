import React from 'react'
import { render } from "@testing-library/react"
import Transacao from "./Transacao"

describe('Veirfy Transaction extract component', ()=> {
    //This test is usefull when you need to verify if something is changed
    //and you not add or verify the tests again to add the new features or values.
    it('Component snapshot should ever be the same', ()=> {
        const {container} = render(<Transacao
            data="08/09/2020"
            tipo="saque"
            valor="20.00"
        />)

        expect(container.firstChild).toMatchSnapshot();
    })
})