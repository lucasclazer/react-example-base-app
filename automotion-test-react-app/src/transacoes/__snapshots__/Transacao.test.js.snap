// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Veirfy Transaction extract component Component snapshot should ever be the same 1`] = `
<div
  class="transacao-container"
>
  <p>
    08/09/2020
  </p>
  <p>
    saque
  </p>
  <p>
    R$ 
    20.00
  </p>
</div>
`;
